project/Dependencies.scala

```diff
@@ -5,24 +5,24 @@ object Dependencies {
+  val `cassandra-driver`         = "com.datastax.cassandra" % "cassandra-driver-core" % "3.11.3"
+  val `play-json`                = "com.typesafe.play"      %% "play-json"                      % "2.10.6"
+  val `play-json-jsoniter`       = "com.evolutiongaming"    %% "play-json-jsoniter"             % "0.10.3"
+  val `executor-tools`           = "com.evolutiongaming"    %% "executor-tools"                 % "1.0.4"
+  val `akka-serialization`       = "com.evolutiongaming"    %% "akka-serialization"             % "1.0.6-SNAPSHOT"
   val `kafka-launcher`           = "com.evolutiongaming"    %% "kafka-launcher"                 % "0.2.0"
-  val `testcontainers-cassandra` = "com.dimafeng"           %% "testcontainers-scala-cassandra" % "0.41.4"
-  val hostname                   = "com.evolutiongaming"    %% "hostname"                       % "0.2.0"
+  val `testcontainers-cassandra` = "com.dimafeng"           %% "testcontainers-scala-cassandra" % "0.40.17"
+  val `testcontainers-kafka` = "com.dimafeng"           %% "testcontainers-scala-kafka" % "0.40.17"
   val scassandra                 = "com.evolutiongaming"    %% "scassandra"                     % "5.2.1"
-  val `cassandra-sync`           = "com.evolutiongaming"    %% "cassandra-sync"                 % "3.0.0"
+  val `cassandra-sync`           = "com.evolutiongaming"    %% "cassandra-sync"                 % "3.0.1-SNAPSHOT"
   val `cats-helper`              = "com.evolutiongaming"    %% "cats-helper"                    % "3.11.0"
-  val random                     = "com.evolutiongaming"    %% "random"                         % "1.0.0"
-  val retry                      = "com.evolutiongaming"    %% "retry"                          % "3.0.1"
-  val sstream                    = "com.evolutiongaming"    %% "sstream"                        % "1.0.1"
-  val skafka                     = "com.evolutiongaming"    %% "skafka"                         % "16.0.0"
+  val random                     = "com.evolution"    %% "random"                         % "1.0.4"
+  val retry                      = "com.evolutiongaming"    %% "retry"                          % "3.1.0"
+  val sstream                    = "com.evolutiongaming"    %% "sstream"                        % "1.0.2"
+  val skafka                     = "com.evolutiongaming"    %% "skafka"                         % "16.4.0"
   val `akka-test-actor`          = "com.evolutiongaming"    %% "akka-test-actor"                % "0.1.0"
-  val scache                     = "com.evolution"          %% "scache"                         % "5.0.0"
-  val `resource-pool`            = "com.evolution"          %% "resource-pool"                  % "1.0.4"
+  val scache                     = "com.evolution"          %% "scache"                         % "5.1.2"
+  val `resource-pool`            = "com.evolution"          %% "resource-pool"                  % "1.0.5-SNAPSHOT"
```

Comment:
I suppose those `*-SNAPSHOT` dependencies are only for local development, right?

Reply:
please take in account that some libraries might bring in some transitively incompatible dependencies - please be careful

Reply:
yes, I have a tree of those dependencies written, there will be more MRs.